networkx/algorithms/isomorphism/demo.py

```diff
@@ -1,12 +1,10 @@
-import random
 import time
 
 import networkx as nx
-from networkx.algorithms.isomorphism.VF2pp import VF2pp
 
 # Graph initialization
-G1 = nx.gnp_random_graph(500, 0.5, 42)
-G2 = nx.gnp_random_graph(500, 0.5, 42)
+G1 = nx.gnp_random_graph(50, 0.6, 42)
+G2 = nx.gnp_random_graph(50, 0.6, 42)
```

Comment:
```suggestion
G1 = nx.gnp_random_graph(50, 0.6, seed=42)
G2 = nx.gnp_random_graph(50, 0.6, seed=42)
```
Just a minor nit. But easier to identify the 3rd argument as the seed without looking up the function. Plus I think we will be shifting to using keyword-required inputs for the seed keywords once the python version we support is large enough.

Reply:
yea agree. This file is going to be removed anyway when we merge to networkx. It's just for experimenting